docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2448,16 +2447,7 @@ E.g., with the following rules:
 ||example.com/secondpath^
 ```
 
-the request to `https://example.com/firstpath` will be blocked before it is sent.
-
-However, `$urltransform` rules will **not be re-applied** in this case to avoid infinite recursion, e.g., with the following rules:
```

Comment:
Tbh, this should be listed as a known issue of CoreLibs implementation.

Reply:
How is avoiding an infinite loop an issue? :)